features/signup.feature

```diff
@@ -15,3 +15,44 @@ Scenario: Sign up with proper credentials
 	And I fill in "user[password_confirmation]" with "hello1"
 	And I press "Sign up"
 	Then I should see "Enter dietary restrictions and preferences here"
+
+
+Scenario: Sign up with improper credentials
```

Comment:
Good tests for sad path scenarios of sign-up process. Although I'm guessing these are implicitly assumed to work because this functionality is provided by Devise? No such thing as too much testing though :)

Reply:
Yeah i'll definitely make sure that i look into the documentation a little more carefully before i start making tests